### DISCUSSION TITLE
key remappings to execute `:set` commands

### DISCUSSION THREAD

**Discussion by mkflan** [helix-editor/helix#9000] [Troubleshooting] [answered]
I recently thought about remapping a few keys to enable and disable LSP inlay hints so they won't get in the way when I don't want them. I set a remapping for disabling inlay hints to test. Unfortunately, the remapping did not work. I experimented more and realized that this did not work for a few other `:set` commands, so I concluded it doesn't work for any `:set` commands. I believe remappings for executing `:set` command should be a feature. I am not sure whether this would be a new feature, or if it is intended to be a feature but is buggy, so I didn't create this issue as an improvement or bug report.

**Comment by the-mikedavis** [comment]
What was the remapping you tried that didn't work? I'm not aware of any bugs with binding `:set` off the top of my head

**Reply by mkflan** [reply]
I tried binding both `:set lsp.display-inlay-hints false` and `:set lsp.display-inlay-hints true`. I binded them by doing `KEY = ":set lsp.display-inlay-hints VAL`. Do I have to instead map a list containing each separate string?

**Reply by David-Else** [reply, marked as answer]
Why not toggle them?
```
[keys.normal.space.t]
i = ":toggle lsp.display-inlay-hints"
```

**Comment by mkflan** [comment]
> Why not toggle them?
> 
> ```
> [keys.normal.space.t]
> i = ":toggle lsp.display-inlay-hints"
> ```

That worked. I wasn't aware of the `:toggle` command.
